src/lib/mina_generators/zkapp_command_generators.ml

```diff
@@ -1069,7 +1078,9 @@ let max_account_updates = 2
 
 let max_token_updates = 2
 
-let gen_zkapp_command_from ?global_slot ?failure
+let gen_zkapp_command_from ?global_slot
+    ?(ignore_sequence_events_precond = false) ?(no_token_accounts = false)
+    ?(limited = false) ?(generate_new_accounts = true) ?failure
```

Comment:
When is `limited` true?

Is there a better name?

Reply:
`limited` means we would not go through the ledger to populate the `account_state_tbl`, we would only work with the existing one.